### PR TITLE
[Docs] Update "customizing menus"

### PR DESCRIPTION
**Pull request by AdamKasp**
| Q               | A
| --------------- | -----
| Branch?         | 1.3
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | fixes 
| License         | MIT

### CODE REVIEW

**Comment by GSadee** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "docs-customization-menu"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

File: docs/customization/menu.rst [hunk 1]
```diff
@@ -274,6 +267,11 @@ listener to the ``sylius.menu.admin.order.show`` event in the ``config/services.
             tags:
                 - { name: kernel.event_listener, event: sylius.menu.admin.order.show, method: addAdminOrderShowMenuItems }
 
+After these two steps your admin panel order menu should look like that, the new item appears at right corner:
```
Comment (Zales0123) [inline docs/customization/menu.rst]: After these two steps, your admin panel order menu should look like that (the new item appears at the right corner):

File: docs/customization/menu.rst [hunk 2]
```diff
@@ -332,6 +327,11 @@ listener to the ``sylius.menu.admin.product.form`` event in the ``config/service
             tags:
                 - { name: kernel.event_listener, event: sylius.menu.admin.product.form, method: addItems }
 
+After these two steps your admin panel product form menu should look like that, the new item appears at the bottom:
```
Comment (Zales0123) [inline docs/customization/menu.rst]: Same here

File: docs/customization/menu.rst [hunk 3]
```diff
@@ -389,6 +386,11 @@ listener to the ``sylius.menu.admin.product_variant.form`` event in the ``config
             tags:
                 - { name: kernel.event_listener, event: sylius.menu.admin.product_variant.form, method: addItems }
 
+After these two steps your admin panel variant menu should look like that, the new item appears at the bottom:
```
Comment (Zales0123) [inline docs/customization/menu.rst]: And here

File: docs/customization/menu.rst [hunk 4]
```diff
@@ -238,19 +235,15 @@ that will let the admin fulfill the order.
...
-            $stateMachine = $event->getStateMachine();
 
-            if ($stateMachine->can(OrderTransitions::TRANSITION_FULFILL)) {
+            if (null !== $order->getId()) {
                 $menu
-                    ->addChild('fulfill', [
-                        'route' => 'sylius_admin_order_fulfill',
+                    ->addChild('ship', [
+                        'route' => 'sylius_admin_order_shipment_ship',
                         'routeParameters' => ['id' => $order->getId()]
```
Comment (Zales0123) [inline docs/customization/menu.rst]: I would also add `->setLabel('sylius.ui.ship')` (without the label is `ship` written lowercase, which looks ugly 😢)

File: docs/customization/menu.rst [hunk 5]
```diff
@@ -19,6 +19,11 @@ You've got six events that you should be subscribing to:
 How to customize Admin Menu?
 ----------------------------
 
+.. tip::
+
+    You can browse the full implementation of these examples on `this GitHub Pull Request.
+    <https://github.com/Sylius/Customizations/pull/16>`_
```
Comment (GSadee) [inline docs/customization/menu.rst]: Incorrect number of PR

**Comment by GSadee** [comment]
Thanks, Adam! :1st_place_medal:

**Pull request merged**